new-site/src/components/ComponentsList.js

```diff
@@ -0,0 +1,283 @@
+      href: '/components/side-navigation',
+    },
+    {
+      name: 'StatusLabel',
+      text: 'Status labels can be used to highlight different statuses or items to the user.',
+      linkboxAriaLabel: 'StatusLabel component',
+      linkAriaLabel: 'Go to the StatusLabel component page',
+      href: '/components/status-label',
+    },
+    {
+      name: 'Stepper',
+      text: 'The stepper helps to see the progression and to navigate between form steps.',
+      linkboxAriaLabel: 'Stepper component',
+      linkAriaLabel: 'Go to the Stepper component page',
+      href: '/components/stepper',
+    },
+    {
+      name: 'Table',
+      text: 'Tables are used to present tabulated data to the user in a consistent manner.',
+      linkboxAriaLabel: 'Table component',
+      linkAriaLabel: 'Go to the Table component page',
+      href: '/components/table',
+    },
+    {
+      name: 'Tabs',
+      text: 'Tabs are used to navigate between contents that have a relation.',
+      linkboxAriaLabel: 'Tabs component',
+      linkAriaLabel: 'Go to the Tabs component page',
+      href: '/components/tabs',
+    },
+    {
+      name: 'Tag',
+      text: 'Tags are used to show attributes of an object or element such as categories.',
+      linkboxAriaLabel: 'Tag component',
+      linkAriaLabel: 'Go to the Tag component page',
+      href: '/components/tag',
+    },
+    {
+      name: 'TextArea',
+      text: 'A text area is an input used for multiline answers.',
+      linkboxAriaLabel: 'TextArea component',
+      linkAriaLabel: 'Go to the TextArea component page',
+      href: '/components/text-area',
+    },
+    {
+      name: 'TextInput',
+      text: 'A text field is an input field that the user can use to input content and data.',
+      linkboxAriaLabel: 'TextInput component',
+      linkAriaLabel: 'Go to the TextInput component page',
+      href: '/components/text-input',
+    },
+    {
+      name: 'TimeInput',
+      text: 'A time input helps the user select a specific time.',
+      linkboxAriaLabel: 'TimeInput component',
+      linkAriaLabel: 'Go to the TimeInput component page',
+      href: '/components/time-input',
+    },
+    {
+      name: 'ToggleButton',
+      text: 'Toggle button allows to switch between two states such as "On" and "Off".',
+      linkboxAriaLabel: 'ToggleButton component',
+      linkAriaLabel: 'Go to the ToggleButton component page',
+      href: '/components/toggle-button',
+    },
+    {
+      name: 'Tooltip',
+      text: 'Tooltips are used to present context or background information to the user.',
+      linkboxAriaLabel: 'Tooltip component',
+      linkAriaLabel: 'Go to the Tooltip component page',
+      href: '/components/tooltip',
+    },
+  ];
+
+  return (
+    <div className="components-list">
+      { componentData.map((component) => {
+        return (
+          <div key={component.name} className="components-list__item">
+            <Linkbox
```

Comment:
Without the images, it is a bit hard to see where the previous row component ends and where the next row component starts. Especially on smaller screens.
Could we add borders as a temporary solution? For example:
<img width="1481" alt="Screenshot 2022-06-06 at 10 28 41" src="https://user-images.githubusercontent.com/1610860/172115601-ef263442-6824-4743-b0db-25d97443961c.png">
<img width="777" alt="Screenshot 2022-06-06 at 10 29 34" src="https://user-images.githubusercontent.com/1610860/172115761-23c65a97-c8c3-4c32-9c6c-1e70e7ca367a.png">